src/ui/components/DevToolsProcessingScreen.tsx

```diff
@@ -25,12 +30,18 @@ export function DevToolsProcessingScreen() {
     }
   }, [processingProgress]);
 
-  let message = "Determining length of replay...";
+  let message = "Processing...";
   let secondaryMessage =
     "This could take a while, depending on the complexity and length of the replay.";
+
   if (showStalledMessage) {
     secondaryMessage = "There may be a problem. This is taking much longer than expected.";
-  } else if (processingProgress != null) {
+  } else if (recording?.duration != null) {
+    const durationLabel = formatEstimatedProcessingDuration(recording?.duration);
+    secondaryMessage = `Based on the length of this Replay, we expect processing will take about ${durationLabel}`;
+  }
+
+  if (processingProgress != null) {
```

Comment:
This is a minor logic change, but I think it makes sense given our desire to show the estimated processing duration.